### ISSUE TITLE
How to mark as SPAM?

### ISSUE THREAD

**Issue by dicarsio** [mesozoic/gmail-yaml-filters#37]
Hi,

Great work with this app.

I was trying to mark something as SPAM, but adding label SPAM does not work and marking not_spam: false also doesn't. Am I missing something or it is not available yet?

Thanks,

C.

**Comment by jretz** [comment]
I don't believe Gmail filters themselves, which are what this tool generates, offer the functionality you want. At the very least, Google doesn't provide a UI for it. If you manually set up a filter in the Gmail web interface, you'll see that there is a checkbox called "Never send it to Spam" but nothing that says anything like "Always send it to Spam".

Setting `not_spam: true` will tick that checkbox. I believe setting `not_spam: false`, or just not specifying `not_spam` at all, indicates that you do not want to tick the "Never send it to Spam" checkbox. Google needs to provide an "Always send it to Spam" filter action before this tool can support it.

For email I never want to see, I use `trash: true`. That doesn't have the benefit of training the spam filter, but it does get all matching emails out of my way.

If you really want the spam filter training benefit, the easiest thing I can think of:
1. create a label (e.g., `to_be_marked_as_spam`)
2. create a filter that sends messages there
3. occasionally go into the label, select everything, and mark as spam
